package/MDAnalysis/coordinates/IMD.py

```diff
@@ -0,0 +1,256 @@
+"""
+IMDReader --- :mod:`MDAnalysis.coordinates.IMD`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`MDAnalysis.coordinates.IMD.IMDReader` is a class that implements the
+`Interactive Molecular Dynamics (IMD) protocol <https://imdclient.readthedocs.io/en/latest/protocol_v3.html>`_ for reading simulation
+data using the IMDClient (see `imdclient <https://github.com/Becksteinlab/imdclient>`_).
```

Comment:
Use intersphinx linking for objects
```suggestion
data using the :class:`~imdclient.IMDClient.IMDClient`).
```